src/dsp_tools/models/propertyclass.py

```diff
@@ -16,12 +16,12 @@ class PropertyClass(Model):
     ROUTE: str = "/v2/ontologies/properties"
 
     _context: Context
-    _id: str
+    _iri: str
     _name: str
     _ontology_id: str
     _superproperties: list[str]
-    _object: str
```

Comment:
`object` is reserved in python. But when renaming it, I had to be consistent and rename `_object` and `_subject`, too.

Reply:
I sometimes use `obj` and `subj` to avoid this problem and keep the variables short. But that's good too